### PR TITLE
dev(defaultLabels): added functionality to add default labels from vi…

### PR DESCRIPTION
**Pull request by printf-twinkle**
…ew organisations page

### Functionality:
Create default labels from view organisation page

### Solution:
Directly click on label icon to add default labels to the team

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
Go to my organisation page and click on the label icon.

### CODE REVIEW

**Review by viv9k — CHANGES_REQUESTED** [review]
We need to trigger api to create default labels for the team where we don't have labels

**Comment by viv9k** [comment]
This is just opening the page, the problem is what about our team that's currently in production.
For that we don't have labels already ready, we need to trigger an API to get it created

File: functions/model/teams/tark/createLabelProperties.js [hunk 1]
```diff
@@ -23,7 +23,13 @@
...
 const { setLabelProperties, getTeam, updateTeamDetails } = require("../lib");
 
-exports.createLableProperties = function(orgDomain, teamName, type, statusLabels, priorityLabels, difficultyLabels) {
+exports.createLabelProperties = function(request, response) {
+    const orgDomain = request.body.data.OrganizationDomain;
+    const teamName = request.body.data.TeamName;
+    const type = request.body.data.Type;
+    const statusLabels = request.body.data.StatusLabels;
+    const priorityLabels = request.body.data.PriorityLabels;
+    const difficultyLabels = request.body.data.DifficultyLabels;
```
Comment (viv9k) [inline functions/model/teams/tark/createLabelProperties.js:32]: This will create a bug into the code.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please create a new instance of the api to make it there.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please fix this.

File: functions/model/teams/tark/createLabelProperties.js (hunk 1 above)
Comment (printf-twinkle) [reply functions/model/teams/tark/createLabelProperties.js:32]: But it is working fine!

File: src/app/body/view-organization-details/teams-card/teams-card.component.ts [hunk 2]
```diff
@@ -65,6 +65,26 @@ export class TeamsCardComponent implements OnInit {
...
+  //     },
+  //     error: (error) => {
+  //       console.error("Error", error);
+  //       this.errorHandlerService.showError = true;
+  //       this.errorHandlerService.getErrorCode(this.componentName, "InternalError","Api");
+  //     },
+  //     complete: () => console.info('Successful ')
+  // });
+  // }
+
```
Comment (viv9k) [inline src/app/body/view-organization-details/teams-card/teams-card.component.ts:87]: remove commented code

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required improvement

File: functions/model/teams/tark/createDefaultLabels.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
+/* eslint-disable linebreak-style */
+/** *********************************************************
+ * Copyright (C) 2022
+ * Worktez
+ * Twinkle Chatterjee (ctwinkle2812@gmail.com)
```
[the remark this answers is not in the document]
Comment (printf-twinkle) [reply functions/model/teams/tark/createDefaultLabels.js:5]: done

File: functions/model/teams/tark/createDefaultLabels.js [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+const { setLabelProperties, getTeam, updateTeamDetails } = require("../lib");
+
+exports.createDefaultLabels = function(request, response) {
+    const orgDomain = request.body.data.OrganizationDomain;
+    const teamName = request.body.data.TeamName;
+    const type = request.body.data.Type;
+    const statusLabels = request.body.data.StatusLabels;
+    const priorityLabels = request.body.data.PriorityLabels;
+    const difficultyLabels = request.body.data.DifficultyLabels;
```
[the remark this answers is not in the document]
Comment (printf-twinkle) [reply functions/model/teams/tark/createDefaultLabels.js:32]: done

File: functions/model/teams/tark/createLabelProperties.js [hunk 5]
```diff
@@ -23,7 +23,7 @@
 
 const { setLabelProperties, getTeam, updateTeamDetails } = require("../lib");
 
-exports.createLableProperties = function(orgDomain, teamName, type, statusLabels, priorityLabels, difficultyLabels) {
+exports.createLabelProperties = function(orgDomain, teamName, type, statusLabels, priorityLabels, difficultyLabels) {
```
Comment (viv9k) [inline functions/model/teams/tark/createLabelProperties.js:26]: Please check and verify the name changes in the places it's used

**Review by viv9k — CHANGES_REQUESTED** [review]
Please check the required comment

File: functions/model/teams/tark/createLabelProperties.js (hunk 5 above)
Comment (printf-twinkle) [reply functions/model/teams/tark/createLabelProperties.js:26]: createLabelProperties is used in createTeam api and createDefaultLabelProperties is used nowhere else.

File: functions/model/teams/tark/createDefaultLabels.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+    createLabelProperties(orgDomain, teamName, type, statusLabels, priorityLabels, difficultyLabels);
```
[the remark this answers is not in the document]
Comment (printf-twinkle) [reply functions/model/teams/tark/createDefaultLabels.js:33]: done but not sure, please check

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**